.github/workflows/icu_ci.yml

```diff
@@ -43,27 +43,6 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: '8'
```

Comment:
I am surprised that this line already said '8' ...

Reply:
Further down in that file we test ICU4J with java-version: '11' and java-version: '16', respectively.